packages/v3/src/esp-entity-table.ts

```diff
@@ -337,28 +337,22 @@ class ActionRenderer {
     entity: entityConfig,
     action: string,
     opt: string,
-    value: number | string,
-    min?: number,
-    max?: number,
+    value: string | number,
+    min?: string | undefined,
+    max?: string | undefined,
     step = 1
   ) {
     if(entity.mode == 1) {
-      const val = Number(value);
-      let stepString = step.toString();
-      let numDecimalPlaces = 0
-      if (stepString.indexOf('.') !== -1) {
-        numDecimalPlaces = stepString.split('.')[1].length;
-      }
       return html`<div class="range">
         <label>${min || 0}</label>
         <input
           type="${entity.mode == 1 ? "number" : "range"}"
           name="${entity.unique_id}"
           id="${entity.unique_id}"
           step="${step}"
-          min="${min || Math.min(0, val)}"
-          max="${max || Math.max(10, val)}"
-          .value="${(val.toFixed(numDecimalPlaces))}"
+          min="${min || Math.min(0, value as number)}"
+          max="${max || Math.max(10, value as number)}"
```

Comment:
Not terribly important, but by keeping `const val = Number(value);` (or `const val = parseFloat(value);`), the type casting `as number` can be removed.